### DISCUSSION TITLE
Changing Button Roles

### DISCUSSION THREAD

**Discussion by DylanBPY** [pybricks/support#1865] [Q&A] [answered]
Hello, i have a program that contains a custom run selector, I want to use the center button to select and stop programs in this run selector.
However, it looks like the center button ends the PyBricks program. Is there a way I could make the Bluetooth button end the PyBricks program that way I can make the center button work with my run selector?

**Comment by dlech** [comment, marked as answer]
I believe this is the function you are looking for: https://docs.pybricks.com/en/stable/hubs/primehub.html#pybricks.hubs.PrimeHub.system.set_stop_button

**Reply by DylanBPY** [reply]
That worked, thanks for the quick response!
